libr/include/r_util/r_annotated_code.h

```diff
@@ -0,0 +1,123 @@
+
+#ifndef R2GHIDRA_ANNOTATEDCODE_H
```

Comment:
change the name of this define, as this is not anymore r2ghidra code.

Reply:
Done